src/v3/src/components/WebAuthNSubmitButton/WebAuthNSubmitButton.tsx

```diff
@@ -30,42 +30,34 @@ const WebAuthNSubmit: UISchemaElementComponent<{
 }> = ({ uischema }) => {
   const { options } = uischema;
 
-  const { setMessages } = useWidgetContext();
+  const { setMessage } = useWidgetContext();
   const onSubmitHandler = useOnSubmit();
-  const { t, i18n } = useTranslation();
   const [waiting, setWaiting] = useState<boolean>(false);
 
-  const waitingIndicatorLabel = t('renderers.waiting.indicator.label');
-  const waitingIndicatorStateLabel = t('renderers.waiting.indicator.state.label');
-
-  const getErrorMessage = (error: Error): string => {
-    const expectedErrorKey = `oie.browser.error.${error.name}`;
-    if (i18n.exists(expectedErrorKey)) {
-      return t(expectedErrorKey);
-    }
-    return error.message;
-  };
+  const waitingIndicatorLabel = loc('renderers.waiting.indicator.label', 'login');
+  const waitingIndicatorStateLabel = loc('renderers.waiting.indicator.state.label', 'login');
 
   const executeNextStep = () => {
     if (options?.showLoadingIndicator) {
       setWaiting(true);
     }
-    setMessages([]);
+    setMessage(undefined);
 
     options?.onClick()
       .then(async (params: IdxActionParams) => {
+        setMessage(undefined);
         onSubmitHandler({
           params,
           includeData: true,
         });
       })
       .catch((error: Error) => {
-        const message = getErrorMessage(error);
-        setMessages([{
+        const message = getMessageFromBrowserError(error);
+        setMessage({
           message,
-          class: MessageType.ERROR,
+          class: 'ERROR',
```

Comment:
Any reason we stopped using `MessageType.ERROR` here?

Reply:
Yes, so what was happening previously was we used the enum here and when it arrived at the component, it could not find the lookup value, and would apply the wrong class. So now, we just use the string instead and the component will perform a lookup using the enum to apply the appropriate class.

I think I will add a type to that class prop to enforce `ERROR | INFO`

Reply:
> I think I will add a type to that class prop to enforce ERROR | INFO

👍

Reply:
So unfortunately because of the type of the message.class prop (string) we cant enforce the type here to only be `ERROR | INFO`